fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java

```diff
@@ -853,6 +1069,87 @@ protected void unprotectedCommitTransaction(TransactionState transactionState,
         }
     }
 
+    protected void unprotectedPrepareTransaction(TransactionState transactionState,
```

Comment:
If have the prepare, can the commit and publish be merged?

Reply:
i don't want to change the original transaction logic in this pr.